go/private/rules/wrappers.bzl

```diff
@@ -20,7 +20,7 @@ load("@io_bazel_rules_go//go/private:rules/cgo.bzl", "setup_cgo_library")
 def go_library_macro(name, srcs=None, embed=[], cgo=False, cdeps=[], copts=[], clinkopts=[], library=None, **kwargs):
   """See go/core.rst#go_library for full documentation."""
   if library:
-    #TODO: print("DEPRECATED: {}//{}:{} : the library attribute is deprecated. Please migrate to embed.".format(native.repository_name(), native.package_name(), name))
+    #TODO: print("\nDEPRECATED: {}//{}:{} : the library attribute is deprecated. Please migrate to embed.".format(native.repository_name(), native.package_name(), name))
```

Comment:
Since this deprecation will be carried over a couple releases, it would probably be good to open an issue and reference it from these TODOs.

Reply:
These are still commented out, I tried to turn them on and then realized they are broken, so I turned them back off but left the \n I added to make them readable, I will be removing the TODO in another cl (hopefully today)